ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -859,6 +875,20 @@ internal class CustomerCenterViewModelImpl(
         }
     }
 
+    private fun computeUserDetailsInfo(
+        customerInfo: CustomerInfo,
+        customerCenterConfigData: CustomerCenterConfigData,
+    ): Triple<String, String?, Boolean> {
+        val originalAppUserId = customerInfo.originalAppUserId ?: ""
+        val originalPurchaseDate = customerInfo.originalPurchaseDate?.let {
+            dateFormatter.format(it, locale)
+        }
+        val shouldShowUserDetailsSection =
+            customerCenterConfigData.support.displayUserDetailsSection &&
+                (BuildConfig.DEBUG || originalPurchaseDate != null)
+        return Triple(originalAppUserId, originalPurchaseDate, shouldShowUserDetailsSection)
```

Comment:
Instead of returning a Triple, I would create a small data class, specially since there are two Strings